src/core/utils.js

```diff
@@ -63,4 +66,59 @@ export class Utils {
         });
         return arr;
     }
+        /**
+         * Check byte unit of measurment
+         * @param {number} n
+         * @param {number} d
+         * @returns number
+         */
+         static roundOffValue(n, d=2) {
```

Comment:
I think we can change its indentation and make it a little descriptive

Reply:
sure!!

Reply:
modified!!